### ISSUE TITLE
Still being supported?

### ISSUE THREAD

**Issue by ccallebs** [peleteiro/progressbar#17]
If not, I'd like to take over development. There are a few core features I'd like to put in.

**Comment by peleteiro** [comment]
Great! What's your email address on http://rubygems.org/?

**Comment by ccallebs** [comment]
@peleteiro cacallebs@gmail.com -- thanks so much!

**Comment by peleteiro** [comment]
Done! https://rubygems.org/gems/progressbar

**Comment by ccallebs** [comment]
@peleteiro Great! My wife just had a baby so it might be a little bit but I'm excited to start contributing!

**Comment by jfelchner** [comment]
@peleteiro I emailed you and asked you if I could take over the `progressbar` name a while back and you said "no"! 😏 

@ccallebs I'd love to be able to snag the gem from you.  I've been waiting for forever to get the `progressbar` name.  [ruby-progressbar](https://github.com/jfelchner/ruby-progressbar) is the de facto progress bar for Ruby (by usage and total downloads) and I'd love to be able to use the simple gem name `progressbar`.

I'm interested to get your feedback on if there are any features in particular that `ruby-progressbar` [doesn't do](https://github.com/jfelchner/ruby-progressbar/wiki).

**Comment by peleteiro** [comment]
@jfelchner are you sure it was me? jose at peleteiro dot net ? Anyway, I'm a big fan of ruby-progressbar and I see no problem on ruby-progress taking over this project If @ccallebs don't mind.

**Comment by jfelchner** [comment]
@peleteiro yes, but it was a loooong time ago. Like three or four years. ☺️

`ruby-progressbar` actually started with a similar code base before I rewrote it. 

Just to be clear on what I would do before you transfer the gem to me:

* Update the gemspec to match `ruby-progressbar` but just with adding @peleteiro to the author info, changing the gem name to `progressbar` and other minor tweaks
* Release a new 1.8 version of `progressbar` with almost the exact same code as `ruby-progressbar`
* The releases from that point forward would be kept in sync. 
* @peleteiro @ccallebs I'll add both your names to the "thank you" section of the `ruby-progressbar` README with a link back to this repo. 
* Add badges for `progressbar`'s gem stats below the badges for `ruby-progressbar` on the README

@peleteiro, if @ccallebs agrees to transfer the gem, I would love it if you'd update the README here with something like:

> This project has been merged with `ruby-progressbar`. Please see that repo for additional release information. 

With a link to the `ruby-progressbar` README. 

Thanks to both of you. 😄

**Comment by ccallebs** [comment]
@peleteiro @jfelchner I don't mind a bit! I took over the gem and then we had our first child, so my hands have been full since then. I over-estimated how much free time I'd have :)

**Comment by jfelchner** [comment]
@ccallebs awesome! I'll email you at the email address on your profile. :)

**Comment by peleteiro** [comment]
@jfelchner Ok. Let's do it.
